### PR TITLE
Change priority to be a number instead of an enum and set a default for pattern

### PR DESCRIPTION
**Pull request by hnn0003**

### CODE REVIEW

**Comment by hnn0003** [comment]
Must also update version of utils!

**Review by hnn0003 — COMMENTED** [review]
Must also update version of utils!

File: packages/saved-views-client/src/models/savedViews/DisplayStyles.ts [hunk 1]
```diff
@@ -702,7 +702,7 @@ export interface PlanarClipMaskProps {
   mode: PlanarClipMaskMode;
   modelIds?: string;
   subCategoryOrElementIds?: string;
-  priority?: PlanarClipMaskPriority;
+  priority?: number;
```
Comment (roluk) [inline packages/saved-views-client/src/models/savedViews/DisplayStyles.ts:705]: Add changelog entry to saved-views-client CHANGELOG.md

**Review by roluk — CHANGES_REQUESTED** [review]

File: packages/saved-views-client/src/models/savedViews/DisplayStyles.ts (hunk 1 above)
Comment (hnn0003) [reply packages/saved-views-client/src/models/savedViews/DisplayStyles.ts:705]: Added changelog for this. :)

**Review by sarah-pontier — APPROVED** [review]

**Review by roluk — APPROVED** [review]

**Pull request merged**